Always enable cc even if the feature is not enabled.

Some Debian architectures need outline asm, and Debian does not ship pre-built
outline asm.

Index: b/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -122,5 +122,5 @@ version = "3.4.0"
 
-[build-dependencies.cc]
+[build-dependencies.cc_dep]
 version = "1.0.68"
-optional = true
+package = "cc"
 
@@ -184,2 +184,3 @@ use-libc = [
 use-libc-auxv = ["libc"]
+cc = []
 
Index: b/build.rs
===================================================================
--- a/build.rs
+++ b/build.rs
@@ -1,3 +1,2 @@
-#[cfg(feature = "cc")]
-use cc::Build;
+use cc_dep::Build;
 use std::env::var;
@@ -164,3 +163,3 @@ fn link_in_librustix_outline(arch: &str,
     // If "cc" is not enabled, use a pre-built library.
-    #[cfg(not(feature = "cc"))]
+    /*#[cfg(not(feature = "cc"))]
     {
@@ -169,3 +168,3 @@ fn link_in_librustix_outline(arch: &str,
         println!("cargo:rustc-link-lib=static={}", name);
-    }
+    }*/
 
@@ -173,3 +172,3 @@ fn link_in_librustix_outline(arch: &str,
     // version, and assert that the pre-built version is checked in.
-    #[cfg(feature = "cc")]
+    //#[cfg(feature = "cc")]
     {
